include/swift/SIL/SILLocation.h

```diff
@@ -552,6 +558,8 @@ class RegularLocation : public SILLocation {
       : SILLocation(L, RegularKind, Implicit) {}
   RegularLocation(FilenameAndLocation *filePos)
     : SILLocation(filePos, RegularKind) {}
+  RegularLocation(FilenameAndLocation *filePos, bool Implicit)
```

Comment:
We should consider merging this with the function above and either update the call sites to explicitly state "explicit" or have a default argument.